### ISSUE TITLE
Lambda expressions that use no block should not span multiple lines

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#516]
When using lambda expressions and they do not have a block as body, those lambda expressions should not span multiple lines. Instead, they should be placed on the same line.

Following example should trigger a violation:
```c#
DoSomething((x, y) =>
                      DoSomethingElse(someData))
```
